lib/dispatcher/client-stats.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
```

Comment:
Follows implementation of PoolStats. Clients however expose a subset of stats. Hence the separate class.

Reply:
I'll put this class in `utils` or so, as the files within the `dispatcher/` dir are mostly mean for classes that shares `dispatcher` interface.

Reply:
Sounds good. Same for the PoolStats then which prior to this change lived here too, right?